src/components/supporters/Supporters/index.js

```diff
@@ -101,6 +95,10 @@ module.exports = React.createClass({
         url={ page.url }
         image={ page.image.large_image_url }
         name={ page.name }
+        hideCharity={ props.hideCharityName }
+        hideEvent={ props.hideEventName }
+        charity={ page.charity_name }
```

Comment:
Can we do the hide tests here? Like `{ props.hideCharityName && page.charity_name}` ?